src/toil/common.py

```diff
@@ -276,7 +277,7 @@ def prepare_restart(self) -> None:
         # exist and that can't safely be re-made.
         self.write_messages = None
 
-    def setOptions(self, options: Namespace) -> None:
+    def setOptions(self, options: Namespace, debug_mute: bool = False) -> None:
```

Comment:
I might call this `quiet`. Also, we should explain what it does if you set it, since it's kind of non-obvious why you would ever need this.

Reply:
Alternately, instead of having logging as an externally-exposed part of this function's contract, we could just not log inside the function, and find where we're calling it where we actually want logging, and add logging *outside* the function in those places.

Otherwise the docstring needs to be something weird like `Creates a config object from the options object and also optionally logs all the options.` and our function no longer has one clearly defined job.